packages/loader/container-loader/src/loader.ts

```diff
@@ -45,14 +45,6 @@ import { IParsedUrl, parseUrl } from "./utils";
 import { pkgVersion } from "./packageVersion";
 import { ProtocolHandlerBuilder } from "./protocol";
 
-function canUseCache(request: IRequest): boolean {
-	if (request.headers === undefined) {
```

Comment:
it might make sense to ship this as off by default first, and then remove it, so we catch any missed use cases before we totally remove the feature which is much harder to compensate for

Reply:
Agreed. We currently assume the default as `true`, so container runtime needs to know about the option for older loaders.